ambari/assets/dashboards/base_dashboard.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
I got this file by creading a dashboard and requesting it through the API. Don't know if that's the right way

Reply:
As long as it was from the screen board endpoint and not the dashboard endpoint. :smile:

Reply:
ok, so nope, it was the dashboard endpoint